source/shader/StarField.h

```diff
@@ -64,6 +64,9 @@ class StarField {
 	Point pos;
 	double baseZoom = 1.;
 
+	double minZoom = .15;
+	double zoomClamp = .25;
```

Comment:
If the interface doesn't contain the values, aren't they zeroed anyway?

Reply:
Yeah.

Reply:
We don't need to initialize them here, then.
```suggestion
	double minZoom;
	double zoomClamp;
```

Reply:
☹️